frontend/tests/utils/helpers.ts

```diff
@@ -407,7 +408,9 @@ export async function takeSnapshotOfMetaTags(page: Page) {
         )
       ).sort();
 
-      expect(JSON.stringify(allMetaTagsHTML)).toMatchSnapshot();
+      expect(JSON.stringify(allMetaTagsHTML)).toMatchSnapshot({
+        name: name + "-seoMetaTags.txt",
```

Comment:
have stable snapshot names to prevent random number increments that don't make sense